torchvision/models/quantization/mobilenetv3.py

```diff
@@ -79,11 +81,12 @@ def _load_weights(
     model: QuantizableMobileNetV3,
     model_url: Optional[str],
     progress: bool,
+    strict: bool = True
 ) -> None:
     if model_url is None:
         raise ValueError("No checkpoint is available for {}".format(arch))
     state_dict = load_state_dict_from_url(model_url, progress=progress)
-    model.load_state_dict(state_dict)
+    model.load_state_dict(state_dict, strict=strict)
```

Comment:
Instead of passing `strict=False` here to be able to get the pre-trained weights loaded, another option would be to implement a `-load_from_state_dict` method in `QuantizableSqueezeExcitation`, adding a `v2` to it and handling the missing fields.

An example on how we did it in the past can be seen in https://github.com/pytorch/vision/blob/a9d710a8ae9df7922981ffc79ec5102a8a927352/torchvision/ops/misc.py#L69-L85

This could be more robust, specially if an user has its own checkpoint based on their finetuned quantized `mobilenetv3`.

Reply:
Hmm, interesting. Thanks for the suggestion. 

Let me provide a bit of context around this. Here I use a cheap workaround of passing `strict=False`. I don't really like this "solution" but I did this for 3 reasons:
1. It helps me avoid introducing a Versioning mechanism before the Model Versioning RFC is properly discussed. 
2. It's unclear to me how the new FX quantization API will affect the `Quantizable*` classes, so I didn't want to change anything but the bare minimum.
3. We are very close to the 0.11 release and I want to ensure no multiple SE layers will be released. I also wanted to avoid introducing complex versioning mechanisms that I won't be able to take back.

But as I really don't like workarounds. Let's chat about your proposal offline and see whether we will implement it now or after the release.

Reply:
I synced with Francisco offline and did a quick investigation to implement the proposal (see https://github.com/pytorch/vision/pull/4487/commits/b491fa26fd7638fed05665fcf40a51f42734122a). Unfortunately it seems that once the patch is applied, the accuracy of the model deteriorates significantly:
`Test:  Acc@1 0.120 Acc@5 0.534`

I'm pretty sure this can be done, but it requires more investigation. I propose to merge the workaround and defer the investigation for the future as this is an implementation detail that can be changed in a BC-manner.